sdk/communication/azure-communication-administration/src/main/java/com/azure/communication/administration/PhoneNumberClient.java

```diff
@@ -479,26 +479,26 @@ public void cancelSearch(String searchId) {
     }
 
     /**
-     * Initiates a search and returns a {@link PhoneNumberSearch} usable by other functions
+     * Initiates a search and returns a {@link PhoneNumberReservation} usable by other functions
      * This function returns a Long Running Operation poller.
-     * 
+     *
      * @param options A {@link CreateSearchOptions} with the search options
-     * @param pollInterval The time our long running operation will keep on polling 
+     * @param pollInterval The time our long running operation will keep on polling
      * until it gets a result from the server
      * @return A {@link SyncPoller} object with the search result
      */
     @ServiceMethod(returns = ReturnType.COLLECTION)
-    public SyncPoller<PhoneNumberSearch, PhoneNumberSearch> beginCreateSearch(
+    public SyncPoller<PhoneNumberReservation, PhoneNumberReservation> beginCreateSearch(
         CreateSearchOptions options, Duration pollInterval) {
```

Comment:
are we changing this to `ReservationOptions` too?

Reply:
Yes it is now changed